src/actions/AuthActions.js

```diff
@@ -1,4 +1,4 @@
-import {Facebook} from 'expo';
+import * as Facebook from 'expo-facebook';
```

Comment:
This is the pattern I was referring to in `package.json`. You'll see it throughout the PR as the new pattern that we need to use to import expo libraries.